persistence/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/protobuf/PersistenceMessageSerializer.scala

```diff
@@ -37,7 +36,7 @@ private[lagom] class PersistenceMessageSerializer(val system: ExtendedActorSyste
   val PersistExceptionManifest          = "D"
   val EnsureActiveManifest              = "E"
 
-  private val emptyByteArray = Array.empty[Byte]
+  Array.empty[Byte]
```

Comment:
Remove the entire line?

Reply:
Good catch. `RemoveUnused` often produces this kind of effects where the assigned `val` is unused but the statement may be side-effectful.